services/ui-src/src/utils/other/time.ts

```diff
@@ -163,3 +163,12 @@ export const calculateRemainingSeconds = (expiresAt?: any) => {
   if (!expiresAt) return 0;
   return moment(expiresAt).diff(moment()) / 1000;
 };
+
+export const displayLongformPeriod = (period: number | undefined) => {
+  const currentYear = new Date().getFullYear();
```

Comment:
We can grab the reportYear from the metadata here as well. No need to calculate the current year!

Reply:
Might be important because if you review an old SAR the current year would be when you view it, not when the SAR Was actually created

Reply:
good point